keras_cv/layers/object_detection/multi_class_non_max_suppression_test.py

```diff
@@ -35,6 +35,7 @@ def decode_predictions_output_shapes():
         bounding_box_format="xyxy",
         from_logits=True,
         max_detections=100,
+        output_ragged=False,
```

Comment:
If we do decide to add this API flexibility, we should test both cases.

Reply:
Removed dense case - I feel its better to only have ragged, and we can add dense if a use case arises.